dataland-frontend/src/components/pages/ChooseFrameworkForDataUpload.vue

```diff
@@ -45,6 +45,12 @@
                 <div :id="dataType + 'Label'" class="col-3 p-3">
                   <h3>{{ humanizeString(dataType) }}</h3>
                   <p>{{ buildSubtitle(humanizeString(dataType)) }}</p>
+                  <p v-if="dataType === DataTypeEnum.P2p">
+                    Click the link to visit the source of the framework:
```

Comment:
Something about this description "Click the link to visit the source of the framework" puts me off. Is this description too much overhead maybe? I feel like the link would be enough. I feel "source of the framework" sounds not entirely correct.  However, if you're perfectly fine with it as is, I accept.

Reply:
I don't have a strong opinion here. Only the link seemed a bit empty when i implemented it. Also iam not sure if only the link is displayed if it is clear that it will lead to the source of the framework and not to some other page on dataland itself.

Reply:
Ok, lets keep it as is for now.

Reply:
Actually, the source of the framework is PwC. How about the base of the framework.

Reply:
But if you have a good idea iam happy to change it.
Maybe somehting like:
Source of the framework: Link

And no other text?

Reply:
Changed